docs/pages/versions/unversioned/expokit/expokit.md

```diff
@@ -80,28 +80,34 @@ ExpoKit's release cycle follows the Expo SDK release cycle. When a new version o
 
 ### iOS
 
-- Open up `ios/Podfile` in your project, and update the `ExpoKit` tag to point at the [release](https://github.com/expo/expo/releases) corresponding to your SDK version. Run `pod update` then `pod install`.
+- Go to https://expo.io/--/api/v2/versions and find the `expokitNpmPackage` key under `sdkVersions.[NEW SDK VERSION]`.
+- Open up `ios/Podfile` in your project, and update the `ExpoKit` tag to point at the release corresponding to your SDK version (found on the link above in `sdkVersions.[NEW SDK VERSION].iosVersion`). 
+- Run `pod update` then `pod install`.
 - Open `ios/your-project/Supporting/EXSDKVersions.plist` in your project and change all the values to the new SDK version.
 
+Nb: If your app is for iOS only, you can run `yarn remove expokit`/ `npm remove expokit`.
```

Comment:
I'm not sure it really makes sense to add this note to the instructions. Even though these files do weigh around 50 MBs, they won't ever get bundled into JS bundle or to iOS app.

Reply:
I see your point: it doesn't hurt to keep it in the package.json because it will have no impact on iOS projects.